src/Microsoft.ServiceFabric.Diagnostics/Telemetry/IMeter.cs

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Diagnostics.Telemetry
+{
+    internal interface IMeter<ValueType>
```

Comment:
Let's add doc comments to all APIs meant to be used by our customers. Normally that means public APIs, but in this case internal types as well.